doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -254,6 +254,31 @@ The device sends converted logs to the Matter controller as a response.
 After the crash data is successfully read, it will be removed and further read attempts will notify the user that there is no available data to read.
 To keep the crash log in the memory after reading it, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REMOVE_CRASH_AFTER_READ` Kconfig option to ``n``.
 
+Network and end user logs
+-------------------------
+
+The diagnostic network and end user logs are saved in the dedicated retained RAM partitions.
+The logs are not removed after reading, but when attempting to write new logs to an already full buffer, the oldest logs are replaced.
+
+The diagnostic network and end user logs are designed to be pushed when requested by the user.
+Usually, it is not convenient to duplicate all logger calls to pass the same information, but using other API.
```

Comment:
```suggestion
This can result in the same information being called by multiple APIs, which is usually not desirable bahavior.
```
Does this wording work?